src/test/ui/auxiliary/legacy-const-generics.rs

```diff
@@ -0,0 +1,6 @@
+#![feature(rustc_attrs)]
+
+#[rustc_legacy_const_generics(1)]
+pub fn foo<const Y: usize>(x: usize, z: usize) -> [usize; 3] {
+    [x, Y, z]
+}
```

Comment:
please also add a test for a function which also has a generic type parameter for one of the runtime arguments